### PR TITLE
Fix compose picker content scrolls behind navbar

### PR DESCRIPTION
**Pull request by michaelkirk-signal**
PTAL @charlesmchen

### CODE REVIEW

File: Signal/src/ViewControllers/NewContactThreadViewController.m [hunk 1]
```diff
@@ -295,6 +283,30 @@ - (void)updateTableContents
 
     __weak NewContactThreadViewController *weakSelf = self;
 
+    if (self.contactsViewHelper.contactsManager.isSystemContactsDenied) {
```
Comment (charlesmchen-signal) [inline Signal/src/ViewControllers/NewContactThreadViewController.m]: How do we update this if the user leaves app to grant/deny access?  Should we reload the table on "become active"?

Comment (michaelkirk-signal) [reply Signal/src/ViewControllers/NewContactThreadViewController.m]: Changing contact permissions kills the app.

Comment (charlesmchen-signal) [reply Signal/src/ViewControllers/NewContactThreadViewController.m]: Can you add a comment to this effect please?

Comment (michaelkirk-signal) [reply Signal/src/ViewControllers/NewContactThreadViewController.m]: sure, done.

**Pull request merged**